scijava-function/templates/main/java/org/scijava/function/Inplaces.vm

```diff
@@ -167,5 +168,17 @@ public final class Inplaces {
 		public int mutablePosition() {
 			return mutablePosition;
 		}
+
```

Comment:
Ahh, this is where Kotlin would be so nice...
```kotlin
data class InplaceInfo(val arity: Int, val mutablePosition: Int);
```
end of class definition :joy:

Reply:
Failing that, you could shorten the class by making the fields `public final` instead of `private`, and then dispense with the accessors.

Reply:
> ```kotlin
> data class InplaceInfo(val arity: Int, val mutablePosition: Int);
> ```

Well, if we used Java 16+, we could get nearly the same syntax with records!

> Failing that, you could shorten the class by making the fields `public final` instead of `private`, and then dispense with the accessors.

Sure, but it's also not a change that I feel like we *need* to make, so not gonna do it here :wink: